addons/web/static/src/views/kanban/kanban_renderer.xml

```diff
@@ -57,13 +57,15 @@
                             </div>
                             <div t-if="(env.isSmall or !group.isFolded) and group.progressBars.length" class="o_kanban_counter position-relative d-flex align-items-center justify-content-between" t-att-class="{ 'opacity-25': !group.count }">
                                 <div class="o_kanban_counter_progress progress w-75 bg-300">
+                                    <t t-set="maxWidth" t-value="100 - Math.max(0,group.progressBars.filter(x => x.count > 0).length - 1)*5"/>
                                     <t t-foreach="group.progressBars" t-as="progressBar" t-key="progressBar.value">
+                                        <t t-set="progressWidth" t-value="(progressBar.count > 0)*Math.max(5, progressBar.count / (group.count or 1) * 100)"/>
```

Comment:
This can be simplified to
`progressBar.count ? Math.max(5, progressBar.count / (group.count) * 100) : 0`

Also, you can probably inline this in the `t-attf-style` attribute to minimize the diff

Reply:
In fact, this is inside a `t-if=progressBar.count`, so you can simply do this
```js
Math.max(5, progressBar.count / (group.count) * 100)
```